src/main/resources/logback.xml

```diff
@@ -35,6 +35,7 @@
     <Logger name="beam.utils.DebugActorWithTimer" level="ERROR" />
     <Logger name="beam.calibration" level="INFO" />
     <Logger name="beam.router.BeamRouter" level="ERROR" />
+    <Logger name="beam.agentsim.agents.ridehail.RideHailAgent " level="DEBUG" />
```

Comment:
This should probably not be merged in for the final.

Reply:
changed to info -> not my code so little worried about deleting it